### PR TITLE
Add recipe for simplecrop model

### PR DESCRIPTION
**Pull request by mcflugen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mcflugen** [comment]
@cpritcha could you please give this a look and make or suggest changes if you see some things that are not right.

In particular,
* Is the license file ok?
* How about the *info* section of *meta.yaml*?
* Are you ok with me listing you (along with myself) as a maintainer of this feedstock?

**Comment by cpritcha** [comment]
Looks good to me! The license and info are correct. I'm happy being listed as a maintainer.

**Comment by mcflugen** [comment]
@cpritcha Thanks for having a look! I've updated it to point to *v0.2* and added a Windows build.

**Comment by mcflugen** [comment]
@conda-forge-admin, please ping team

Ready for review. Thanks!

File: recipes/simplecrop/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+mkdir src/_build
+cd src/_build
+
+cmake ../ -DCMAKE_INSTALL_PREFIX=$PREFIX
+make all install
```
Comment (ocefpaf) [inline recipes/simplecrop/build.sh:7]: Is there a make check/test or ctest that we can run here?

Comment (mcflugen) [reply recipes/simplecrop/build.sh:7]: The only test I am aware of, unfortunately, is the smoke test I run in the tests section of *meta.yaml*, which just runs *simplecrop* on some example data files.

**Pull request merged**